test/IoC.Test/UnitTest/TestClass.cs

```diff
@@ -0,0 +1,8 @@
+using System;
+
+namespace IoC.Test.UnitTest;
+
+public class TestClass
```

Comment:
I don't you use existing .net classes like dictionaries, lists or sets, I don't think this is necessary at all